lib/shared/config/merge-configs.js

```diff
@@ -5,6 +5,7 @@ var path = require('path');
 
 function mergeConfigs(env) {
   var cfg = {};
+  /* istanbul ignore if */
```

Comment:
Why is this needed now?

Reply:
For increasing coverage. It's impossible to test a config file in user home.